main.go

```diff
@@ -136,7 +156,20 @@ func normalize(this js.Value, args []js.Value) interface{} {
 			}
 			if !nameMap.IsUndefined() {
 				if _, err := UpdateTableNames(ast, func(name string) (string, bool) {
-					value := nameMap.Get(name)
+					// take the name and see if it's captured by any of our enclosure characters
+					// if so, map what's inside the enclosure, if not try to map the original name
+					var value js.Value
+					_name, enclosure, isEnclosed := getEnclosedName(name)
+
+					if isEnclosed {
+						value = nameMap.Get(_name)
+						if value.IsUndefined() {
+							return "", false
+						}
+						return enclosure.open + value.String() + enclosure.close, true
+					}
```

Comment:
Take the name out of the enclosure and try to do the mapping and put the name back in.